Explorer/Assets/Scripts/SceneRunner/PortableExperienceSceneFacade.cs

```diff
@@ -0,0 +1,204 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using Microsoft.ClearScript;
+using SceneRunner.Scene;
+using SceneRunner.Scene.ExceptionsHandling;
+using SceneRuntime;
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using Utility.Multithreading;
+
+namespace SceneRunner
+{
+    public class PortableExperienceSceneFacade : ISceneFacade
```

Comment:
Many methods are copied from `SceneFacade`, introduce a base class to keep common logic there, don't create duplicates

Reply:
roger roger